src/test/acceptance/features/internationalisation.feature

```diff
@@ -4,10 +4,10 @@ Feature: Translations
   I want to view the site in my preferred language
 
   Scenario:
-    When I have entered my details up to the enter national insurance page with Welsh language selected
-    Then the enter national insurance page is in Welsh
+    When I have completed the first step of the application with Welsh language selected
```

Comment:
I have renamed the scenarios in this file to be more generic to avoid constantly renaming as new steps are added to the start of the application. In my opinion the names of the pages are not important in this feature, just that a sequence of pages are loaded in the correct language.